README.md

```diff
@@ -53,6 +53,9 @@ Image Replacement APIs:
 
 > Gemojione.image_url_for_name('heart')
 => "http://localhost:3000/assets/emoji/2764.png"
+
+> Gemojione.replace_ascii_moji_with_images "I <3 Emoji"
+=> "I <img alt=\"❤\" class=\"emoji\" src=\"http://localhost:3000/assets/emoji/2764.png\"> Emoji"
```

Comment:
I believe Ruby standard are to use parenthesis when method isn't taking any arguments. And i've been following this rules from years :) will add parenthesis for this repo now.